scripts/worker.py

```diff
@@ -144,10 +144,14 @@ def get_model(args, gpu, rank):
                                 world_size=args.world_size, rank=rank)
 
     # Why is model & optimizer built in spawned function?
+    resume = False
+    if args.weights_path is not None:
```

Comment:
`resume = (args.weights_path is not None)` should do it

this resume functionality is also going to be fixed in another PR right?

Reply:
yes, created an issue for this  #130

Reply:
Perfect, thanks!